src/nodes/fmod_event_emitter.h

```diff
@@ -330,15 +311,18 @@ namespace godot {
     }
 
     template<class Derived, class NodeType>
-    void FmodEventEmitter<Derived, NodeType>::preload_event() const {
```

Comment:
Why const is removed here ?
I assume is_one_shot method should be const as it is a getter.

Reply:
I have to remove the const, because I set the _is_one_shot variable there. I can't use the _event variable directly to get the one_shot property in its getter because Godot cache default values with dummy instance (and in such case _event would be null, leading to crash).